src/constants.ts

```diff
@@ -15,3 +15,8 @@ export enum MetaMetricsEventName {
 export enum MetaMetricsEventCategory {
   Transactions = 'Transactions',
 }
+
+export enum NetworkClientId {
```

Comment:
Does this already exist in the `NetworkController` since it generates the network client IDs?